python_modules/libraries/dagster-looker/dagster_looker_tests/api/test_build_defs.py

```diff
@@ -195,14 +195,13 @@ def test_build_defs_with_pdts(
 def test_custom_asset_specs(
     looker_resource: LookerResource, looker_instance_data_mocks: responses.RequestsMock
 ) -> None:
-    expected_metadata = {"custom": "metadata"}
-
     class CustomDagsterLookerApiTranslator(DagsterLookerApiTranslator):
-        def get_asset_key(self, looker_structure: LookerStructureData) -> AssetKey:
-            return super().get_asset_key(looker_structure).with_prefix("my_prefix")
-
         def get_asset_spec(self, looker_structure: LookerStructureData) -> AssetSpec:
-            return super().get_asset_spec(looker_structure)._replace(metadata=expected_metadata)
+            default_spec = super().get_asset_spec(looker_structure)
+            return default_spec.replace_attributes(
```

Comment:
nit: this would probably be easier done as merge_attributes, which will automatically merge the metadata dictionary

Reply:
sweet

Reply:
Updated in [47484bf](https://github.com/dagster-io/dagster/pull/26048/commits/47484bf45d0bda86ff17e9400fab311717542840)